src/operator/subgraph/dnnl/dnnl_subgraph_base-inl.h

```diff
@@ -31,7 +31,7 @@ static inline bool SupportDNNLAttr(const std::shared_ptr<NodeAttr>& node_attr) {
     return (node_attr->dispatch_mode == DispatchMode::kFComputeEx) &&
            (node_attr->itype[0] == mshadow::kFloat32 ||
             node_attr->itype[0] == mshadow::kBfloat16) &&
-           (ndim == 1 || ndim == 2 || ndim == 4 || ndim == 5);
+           (ndim >= 1 && ndim <= 5);
```

Comment:
Are you sure `ndim == 3` is fine as well?

Reply:
Yes, i don't see any reason why 3 should be ommited here - probably someone missed it or there was some kind of not noted bug. All tests are passing so I'm assuming it's ok